src/Lib/Canvas.vala

```diff
@@ -50,12 +53,13 @@ public class Akira.Lib.Canvas : Goo.Canvas {
     public InsertType? insert_type { get; set; }
 
     public void set_cursor_by_edit_mode () {
-      if (_edit_mode == EditMode.MODE_SELECTION) {
-        set_cursor (Gdk.CursorType.ARROW);
-      } else {
-        set_cursor (Gdk.CursorType.CROSSHAIR);
-      }
+        if (_edit_mode == EditMode.MODE_SELECTION) {
+            set_cursor (Gdk.CursorType.ARROW);
+        } else {
+            set_cursor (Gdk.CursorType.CROSSHAIR);
+        }
```

Comment:
We should visually show when the user enables the drag action to pan.
Let's change the cursor to the `grab` when holding spacebar, and the `grabbing` when clicking:
https://developer.gnome.org/gdk3/stable/gdk3-Cursors.html

Reply:
While reading the doc you provided, I discovered that the suggested way of changing cursor is by using `Gdk.Cursor.from_name (string)` rather than the `Gdk.Cursor.for_display (Gdk.CursorType)`. The difference is in the way in which we choose the cursor: in the former, you need to type the cursor name, in the latter the CursorType, which is an enum of not-so-well-supported X cursor types. Since `grab` and `grabbing` were not present in the `Gdk.CursorType` enum too, I refactored the code to make use of the `from_name` construct. Everything works the same as before, the only difference is the rotation cursor, I wasn't able to find something similar to the previous one. Maybe we can use a custom cursor.